documentation/4.0/content/managing-domains/choosing-a-model/_index.md

```diff
@@ -13,29 +13,29 @@ This document describes the domain home source types for deploying a domain, and
 
 When using the operator to start WebLogic Server instances from a domain, you have the choice of the following WebLogic domain home source types:
 
- - **[Domain in PV]({{< relref "/samples/domains/domain-home-on-pv/_index.md" >}})**:
-   - Set the domain resource `domain.spec.domainHomeSourceType` attribute to `PersistentVolume`.
-   - Supply a WebLogic installation in an image and supply WebLogic configuration as a domain home in a persistent volume.
-   - Supply WebLogic applications in the persistent volume.
-   - Mutate WebLogic configuration using WLST, the WebLogic Server Administration Console, or [configuration overrides]({{< relref "/managing-domains/configoverrides/_index.md" >}}) supplied in a Kubernetes ConfigMap.
-
- - **[Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}})**:
-   - Set the domain resource `domain.spec.domainHomeSourceType` attribute to `Image`.
-   - Supply a WebLogic installation in an image and supply WebLogic configuration as a domain home layered on this image.
-   - Supply WebLogic applications layered on the installation image.
-   - Mutate WebLogic configuration by supplying a new image and rolling, or by configuration overrides supplied in a Kubernetes ConfigMap.
-
  - **[Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}})**:
    - Set the domain resource `domain.spec.domainHomeSourceType` attribute to `FromModel`.
    - Supply a WebLogic installation in an image and supply WebLogic configuration in one of three ways:
-     - As WebLogic Deployment Tool (WDT) model YAML file layered on the WebLogic installation image.
      - As WDT model YAML file supplied in separate [auxiliary images]({{< relref "/managing-domains/model-in-image/auxiliary-images.md" >}}).
+     - As WebLogic Deployment Tool (WDT) model YAML file layered on the WebLogic installation image.
      - As WDT model YAML file in a Kubernetes ConfigMap.
    - Supply WebLogic applications in one of two ways:
-     - Layered on the installation image.
      - In auxiliary images.
+     - Layered on the installation image.
    - Mutate WebLogic configuration by supplying a new image and rolling, or [model updates]({{< relref "/managing-domains/model-in-image/runtime-updates.md" >}}) supplied in a Kubernetes ConfigMap.
 
+ - **[Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}})**:
+   - Set the domain resource `domain.spec.domainHomeSourceType` attribute to `Image`.
+   - Supply a WebLogic installation in an image and supply WebLogic configuration as a domain home layered on this image.
```

Comment:
I will make your recommended change, but not we're using 'a' instead of 'the' throughout for other artifacts in this write up. For example, we are using 'a WebLogic installation', 'an image' instead of 'the image', and 'a domain home' instead of 'the domain home', etc, etc.

Reply:
Ditto for your other 'the' recommendations in this section...

Reply:
@tbarnes-us That's ok, use "a" but you were missing the article entirely.

Reply:
Use an article, whichever one you decide is correct.

Reply:
done